src/javascript/app/pages/user/new_account/real_acc_opening.js

```diff
@@ -19,6 +19,9 @@ const RealAccOpening = (() => {
                     State.getResponse('authorize.upgradeable_landing_companies').indexOf('costarica') !== -1) {
                     getElementById('risk_disclaimer').setVisibility(1);
                 }
+                if (/gb/.test(Client.get('residence'))) {
```

Comment:
do we want to check that client is also having status unwelcome instead of assuming?